CesiumGltf/include/CesiumGltf/FeatureTexturePropertyView.h

```diff
@@ -0,0 +1,208 @@
+#pragma once
+
+#include "ClassProperty.h"
+#include "ExtensionModelExtFeatureMetadata.h"
+#include "FeatureTexture.h"
+#include "Image.h"
+#include "ImageCesium.h"
+#include "Model.h"
+#include "Sampler.h"
+#include "Texture.h"
+#include "TextureAccessor.h"
+#include "TextureInfo.h"
+
+#include <cassert>
+#include <cstdint>
+#include <variant>
+
+namespace CesiumGltf {
+
+enum class FeatureTexturePropertyViewStatus {
+  Valid,
+  InvalidUninitialized,
+  InvalidTextureIndex,
+  InvalidTextureSamplerIndex,
+  InvalidTextureSourceIndex,
+  InvalidChannelsString
+};
+
+enum class FeatureTexturePropertyComponentType {
+  Uint8
+  // TODO: add more types. Currently this is the only one outputted by stb,
+  // so change stb call to output more of the original types.
```

Comment:
Even if STB only outputs images in certain formats, a user could construct a glTF with textures in alternate formats, right? We should support them, unless it creates a lot of complexity.